app/src/main/res/values/strings.xml

```diff
@@ -2,16 +2,16 @@
 <resources>
 
     <!-- Applications list -->
-    <string name="info_applications_list_refreshed">Applications list refreshed</string>
-    <string name="button_refresh_list">Refresh the applications list</string>

```

Comment:
Plural error

Reply:
Same than above, could it be "Refresh apps list"?

Reply:
It could be "Refresh the list of apps" or (technically)
"Refresh the list on which the apps reside"

Reply:
Ok, I would prefer "Refresh the list of apps" in this case.
(The more technical alternative is a bit long, even for me :joy:)